i3-settings-qubes/qubes-i3status

```diff
@@ -9,7 +9,7 @@ json() {
 }
 
 status_net() {
-    local netvms=$(qvm-ls | awk -F "|" '$5 ~ /.*Net.*/ {match($1, /.*{(.*)}.*/, arr); print arr[1]}')
+    local netvms=$(qvm-ls --raw-data --fields FLAGS 2>/dev/null | grep '...N....' | wc -l)
```

Comment:
The loop below expect list of netvms, not count of them. Also, there is `--no-spinner`.

Reply:
like this?